docs/guide/options/README.md

```diff
@@ -204,23 +209,85 @@ export const mutations = {
   onSuccessMutation: (state, { authUser, claims }) => {
```

Comment:
rename to `ON_AUTH_STATE_CHANGED_MUTATION`

Reply:
Done (to be committed)